docker-compose.yml

```diff
@@ -11,6 +11,7 @@ services:
     volumes:
       - ./backend/code:/code
     env_file: .env
+    links: [postgres]
 
   postgres:
     image: postgres:9.4
```

Comment:
@noemiko @firemark hm, jest jakiś powód dla którego jest tutaj stary postgres? ;)

Reply:
Żaden. Nie jest to w tej chwili ważne jak aplikacja jeszcze nie działa tak jak powinna (ficzery się robią)

Reply:
@firemark dodałem sekcje issues w tym repo, może ktoś się podzieli, to chętnie pomogę przy jakimś feature :)

Reply:
@noemiko  plase be aware, that upgrading postgres will need more effort than just "change version in docker-compose.yml file". It is better to use newest postgres here.